zeroconf/const.py

```diff
@@ -44,10 +43,7 @@
 # Some DNS constants
 
 _MDNS_ADDR = '224.0.0.251'
-_MDNS_ADDR_BYTES = socket.inet_aton(_MDNS_ADDR)
 _MDNS_ADDR6 = 'ff02::fb'
-with contextlib.suppress(OSError):  # can't use AF_INET6, IPv6 is disabled
-    _MDNS_ADDR6_BYTES = socket.inet_pton(socket.AF_INET6, _MDNS_ADDR6)
```

Comment:
Functionally the change looks good 

I want to do some git archeology tomorrow to see if the reason for this being a constant can be determined in case we are both missing something here.

Reply:
I found these, maybe they'll be useful:
* first commit with the constant 3d5787b8c5a92304b70c04f48dc7d5cec8d9aac8
* removed from public API in #563

Reply:
Wow, thanks for finding https://github.com/jstasiak/python-zeroconf/commit/3d5787b8c5a92304b70c04f48dc7d5cec8d9aac8

It looks like this change should be fine as it doesn't strictly need to be a constant.